src/androidTest/java/com/couchbase/lite/replicator/ReplicationTest.java

```diff
@@ -1401,6 +1401,9 @@ public void testRemoteConflictResolution() throws Exception {
             runReplication(pullReplication);
             assertNull(pullReplication.getLastError());
 
+            // With the faster device, stop event is fired before mock server receive request.
+            try { Thread.sleep(1000); } catch (Exception ex) { }
```

Comment:
Doesn't the replicator need to wait for responses from the Mock Server after sending any requests? Could you explain a bit more?